Mon, 13 Jan 2020 16:12:04 - INFO - Connect to DB
Mon, 13 Jan 2020 16:12:05 - INFO - TRIZMA DB import
Mon, 13 Jan 2020 16:12:05 - INFO - 123TV DB import
Mon, 13 Jan 2020 16:12:10 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 491727715486, for agent id: 3122, with error:Missing element acd_id (importCallMetaData.acd_id)
Mon, 13 Jan 2020 16:15:03 - INFO - Connect to DB
Mon, 13 Jan 2020 16:15:03 - INFO - TRIZMA DB import
Mon, 13 Jan 2020 16:15:03 - INFO - 123TV DB import
Mon, 13 Jan 2020 16:15:08 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 49222510330, for agent id: 3019, with error:Missing element acd_id (importCallMetaData.acd_id)
Mon, 13 Jan 2020 16:33:01 - INFO - Connect to DB
Mon, 13 Jan 2020 16:33:01 - INFO - TRIZMA DB import
Mon, 13 Jan 2020 16:33:01 - INFO - 123TV DB import
Mon, 13 Jan 2020 16:33:06 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 499285968557, for agent id: 3122, with error:Missing element acd_id (importCallMetaData.acd_id)
Mon, 13 Jan 2020 16:50:01 - INFO - Connect to DB
Mon, 13 Jan 2020 16:50:01 - INFO - TRIZMA DB import
Mon, 13 Jan 2020 16:50:01 - INFO - 123TV DB import
Mon, 13 Jan 2020 16:50:06 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 4971508104659, for agent id: 3117, with error:Missing element acd_id (importCallMetaData.acd_id)
Mon, 13 Jan 2020 16:54:54 - INFO - Connect to DB
Mon, 13 Jan 2020 16:54:54 - ERROR - Exception on /records [GET]
Traceback (most recent call last):
  File "/home/python_project/hello.py", line 81, in get
    1)).fetchone()[0]
TypeError: 'NoneType' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python3.6/dist-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/python_project/hello.py", line 90, in get
    + str(e))
UnboundLocalError: local variable 'agentUsername' referenced before assignment
Mon, 13 Jan 2020 17:37:52 - INFO - Connect to DB
Mon, 13 Jan 2020 17:37:52 - INFO - TRIZMA DB import
Mon, 13 Jan 2020 17:37:52 - INFO - 123TV DB import
Mon, 13 Jan 2020 17:37:57 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 496940585195, for agent id: 3019, with error:Missing element acd_id (importCallMetaData.acd_id)
Mon, 13 Jan 2020 18:17:33 - INFO - Connect to DB
Mon, 13 Jan 2020 18:17:33 - INFO - TRIZMA DB import
Mon, 13 Jan 2020 18:17:33 - INFO - 123TV DB import
Mon, 13 Jan 2020 18:17:38 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 49234522105, for agent id: 3067, with error:Missing element acd_id (importCallMetaData.acd_id)
Tue, 14 Jan 2020 07:22:57 - INFO - Connect to DB
Tue, 14 Jan 2020 07:22:57 - INFO - TRIZMA DB import
Tue, 14 Jan 2020 07:22:57 - INFO - 123TV DB import
Tue, 14 Jan 2020 07:23:02 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 4923597275, for agent id: 3098, with error:Missing element acd_id (importCallMetaData.acd_id)
Tue, 14 Jan 2020 07:28:46 - INFO - Connect to DB
Tue, 14 Jan 2020 07:28:46 - INFO - TRIZMA DB import
Tue, 14 Jan 2020 07:28:46 - INFO - 123TV DB import
Tue, 14 Jan 2020 07:28:51 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 491789729845, for agent id: 3181, with error:Missing element acd_id (importCallMetaData.acd_id)
Tue, 14 Jan 2020 07:33:08 - INFO - Connect to DB
Tue, 14 Jan 2020 07:33:08 - INFO - TRIZMA DB import
Tue, 14 Jan 2020 07:33:08 - INFO - 123TV DB import
Tue, 14 Jan 2020 07:33:13 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 49407968225, for agent id: 3098, with error:Missing element acd_id (importCallMetaData.acd_id)
Tue, 14 Jan 2020 07:35:40 - INFO - Connect to DB
Tue, 14 Jan 2020 07:35:40 - INFO - TRIZMA DB import
Tue, 14 Jan 2020 07:35:40 - INFO - 123TV DB import
Tue, 14 Jan 2020 07:35:45 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 491709965573, for agent id: 3098, with error:Missing element acd_id (importCallMetaData.acd_id)
Tue, 14 Jan 2020 07:41:27 - INFO - Connect to DB
Tue, 14 Jan 2020 07:41:27 - INFO - TRIZMA DB import
Tue, 14 Jan 2020 07:41:27 - INFO - 123TV DB import
Tue, 14 Jan 2020 07:41:32 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 491624060851, for agent id: 3098, with error:Missing element acd_id (importCallMetaData.acd_id)
Tue, 14 Jan 2020 07:52:04 - INFO - Connect to DB
Tue, 14 Jan 2020 07:52:04 - INFO - TRIZMA DB import
Tue, 14 Jan 2020 07:52:04 - INFO - 123TV DB import
Tue, 14 Jan 2020 07:52:09 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 493613735407, for agent id: 3098, with error:Missing element acd_id (importCallMetaData.acd_id)
Tue, 14 Jan 2020 07:53:35 - INFO - Connect to DB
Tue, 14 Jan 2020 07:53:35 - INFO - TRIZMA DB import
Tue, 14 Jan 2020 07:53:35 - INFO - 123TV DB import
Tue, 14 Jan 2020 07:53:40 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 493571921405, for agent id: 3098, with error:Missing element acd_id (importCallMetaData.acd_id)
Tue, 14 Jan 2020 08:15:39 - INFO - Connect to DB
Tue, 14 Jan 2020 08:15:39 - INFO - TRIZMA DB import
Tue, 14 Jan 2020 08:15:39 - INFO - 123TV DB import
Tue, 14 Jan 2020 08:15:44 - INFO - Failed on 123TV DB import at: 2020-01-13_16-04-16, for following calling party: 491624395307, for agent id: 3098, with error:Missing element acd_id (importCallMetaData.acd_id)
